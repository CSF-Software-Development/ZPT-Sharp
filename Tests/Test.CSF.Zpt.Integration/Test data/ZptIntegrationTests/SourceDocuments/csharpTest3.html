<html tal:define="ref1 csassembly:Test.CSF.Zpt.Integration.dll">
<head>
<title>CSharp Linq test</title>
</head>
<body>
<p tal:define="items options/enumerableItems;
               itemsType cstype:items ICollection&lt;object&gt;;
               scg csnamespace:System.Collections.Generic;
               linq csnamespace:System.Linq;">
  Here is a sequence created using Linq:
  <span tal:content="csharp:String.Join(&quot;,&quot;, items.Select(x => x.ToString()))">contents here</span>
</p>
</body>
</html>