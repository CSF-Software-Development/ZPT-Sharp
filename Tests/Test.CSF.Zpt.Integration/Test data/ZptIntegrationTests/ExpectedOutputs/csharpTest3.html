<html>
<head>
<title>CSharp Linq test</title>
</head>
<body>
<p>
  Here is a sequence created using Linq:
  <span>one,two,three,four,five</span>
</p>
</body>
</html>